pkg/driver/efs_watch_dog.go

```diff
@@ -39,22 +95,43 @@ type execWatchdog struct {
 	execArg []string
 	// the cmd that is running
 	cmd *exec.Cmd
+	// efs-utils config file path
+	efsUtilsCfgPath string
 	// stopCh indicates if it should be stopped
 	stopCh chan struct{}
 
 	mu sync.Mutex
 }
 
-func newExecWatchdog(cmd string, arg ...string) Watchdog {
+func newExecWatchdog(efsUtilsCfgPath, cmd string, arg ...string) Watchdog {
 	return &execWatchdog{
-		execCmd: cmd,
-		execArg: arg,
-		stopCh:  make(chan struct{}),
+		efsUtilsCfgPath: efsUtilsCfgPath,
+		execCmd:         cmd,
+		execArg:         arg,
+		stopCh:          make(chan struct{}),
 	}
 }
 
-func (w *execWatchdog) start() {
+func (w *execWatchdog) start() error {
+	if err := w.updateConfig(); err != nil {
+		return err
+	}
+
 	go w.runLoop(w.stopCh)
+
+	return nil
+}
+
+func (w *execWatchdog) updateConfig() error {
+	efsUtilsConfig := template.Must(template.New("efs-utils-config").Parse(efsUtilsConfigTemplate))
+	f, err := os.Create(w.efsUtilsCfgPath)
```

Comment:
do you need to close the file after write finishes?

Reply:
Nice catch! We should definitely close file after we are done with it to avoid having too many open files. 

Fixed.